### PR TITLE
dom-event-delegation - Ray Le

### PR DESCRIPTION
**Pull request by lol-ray**
CXu5uFtfhX

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', clickTask);
+
+function clickTask(event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+  console.log('closest .task-list-item', event.target.closest('.task-list-item'));
```
Comment (Zhiwutian) [inline dom-event-delegation/main.js:7]: make sure that you are checking to confirm that the element you clicked on was a button. That is a necessary check!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
almost there, make sure to confirm a button was clicked before removing the element

**Comment by lol-ray** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**